torch/csrc/jit/frontend/ir_emitter.cpp

```diff
@@ -2785,7 +2789,13 @@ struct to_ir {
     // If `kwargs` is an empty dict, users are allowed to not pass `kwargs`.
     // If `args` and `kwargs` are an empty tuple and an empty dict,
     // respectively, users are allowed to not pass `args` and `kwargs`.
-    TreeList args_kwargs_trees(input_trees.begin() + 2, input_trees.end());
+
+    TreeList args_kwargs_trees(
+        input_trees.begin() + 2,
+        apply.inputs().size() == rpcMaxInputs ? input_trees.end() - 1 : input_trees.end());
+
+    // Parse timeout, if applicable
+    Value* timeout = apply.inputs().size() == rpcMaxInputs ? emitExpr(Expr(input_trees[4])) : nullptr;
```

Comment:
I think you don't need to change this part.

You just need to change `args_kwargs_trees` to `args_kwargs_timeout_trees`.

Reply:
Good point!